src/Unused/CLI/Views/SearchResult.hs

```diff
@@ -72,8 +79,8 @@ printMatches cf r ms =
         putStr $ "  " ++ removalReason r
         putStr "\n"
   where
-    printTerm = cfPrintTerm cf
-    printPath = cfPrintPath cf
-    printNumber = cfPrintNumber cf
     termColor = likelihoodColor . rLikelihood . trRemoval
     removalReason = rReason . trRemoval
+
+liftReaderT :: m a -> ReaderT r m a
```

Comment:
You shouldn't need this ... `liftIO` does this for you ... imported from `Control.Monad.IO.Class`
http://haddock.stackage.org/lts-5.11/transformers-0.4.2.0/Control-Monad-IO-Class.html